server/athenian/api/align/queries/metrics.py

```diff
@@ -80,8 +80,11 @@ async def resolve_metrics_current_values(
     )
     time_interval = _parse_time_interval(params)
     teams_flat = flatten_teams(team_rows)
+    if (repositories := params.get(MetricParamsFields.repositories)) is not None:
+        repositories = tuple(RepositoryName.from_prefixed(r).unprefixed for r in repositories)
```

Comment:
We must check the repository access using `AccessChecker`. `acc_id` is our final line of defense, we shouldn't overrely on it.

Reply:
added